examples/tuning/PID_controller_tuner.py

```diff
@@ -152,7 +152,10 @@ def __init__(self, pid_gui, scf):
         self.cf.param.add_update_callback(
             group='posCtlPid', name='zKd', cb=self.param_updated_callback_Kd)
         self.cf.param.add_update_callback(
-            group='posCtlPid', name='xyVelMax',
+            group='posCtlPid', name='xVelMax',
```

Comment:
Actually this will still fail as the posCtlPid.xyVelMax parameter is being used in multiple places, when requesting the parameter and updating the value. So those need to be changed as well.

Reply:
doh! thanks!

Pushed a new commit